portfolio/src/main/webapp/index.html

```diff
@@ -63,15 +63,23 @@ <h1 id="name">Omar Montiel's Portfolio</h1>
       <!-- Contact Section -->
       <div id="contact" class="section">
         <a href="resume.pdf" target="_blank">Resume</a>
-        <!-- Message Input Form -->
+        <!-- Comment Input Form -->
         <form action="/data" method="POST">
-          <p>Leave a message:</p>
+          <p>Leave a comment:</p>
           <textarea name="text-input">Say hi!</textarea>
-          <br/><br/>
           <input type="submit" />
         </form>
-        <button onclick="getMessage()">Try Fetch</button> <!-- Fetch message from Data Servlet -->
-        <div id="message-container"></div> <!-- Display message from Data Servlet -->
+        <br>
+        <!-- Comment Output Form -->
+        <form>
+            <label for="quantity">How many comments do you want displayed?</label>
+            <input type="number" id="quantity" name="quantity" min="0">
+            <input type="button" onclick="getComments()" value="Display Comments">
+        </form>
+        <div id="comment-container"></div> <!-- Display comments from DataServlet -->
+        <br>
```

Comment:
Optional - Prefer using css for spacing rather than <br>

Reply:
Noted and done!